components/LivePage.tsx

```diff
@@ -34,7 +35,7 @@ export default function LivePage({
 
       {
         // Track only managed pages
-        page && <LiveAnalytics page={page} site={site} />
+        page && <LiveAnalytics page={page} flags={flags as Flags} site={site} />
```

Comment:
Hey, all pages will have a flag, right? So, it makes sense to me.
Otherwise, LiveAnalytics could handle flags not defined.

```diff
// at liveanatytics.tsx
- Object.keys(flags).map((key) => `flag_${key}: true`).join(",")
+ Object.keys(flags || {}).map((key) => `flag_${key}: true`).join(",") 
```